io.github.wivrn.wivrn.yml

```diff
@@ -0,0 +1,209 @@
+id: io.github.wivrn.wivrn
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: launch-wivrn-server
+
+add-extensions:
+  org.freedesktop.Platform.VAAPI.Intel:
+    version: '23.08'
+    directory: lib/dri
+    add-ld-path: lib
+    subdirectories: true
+    no-autodownload: true
+    autodelete: true
+cleanup-commands:
+  - mkdir -p ${FLATPAK_DEST}/lib/dri
```

Comment:
Why is this here? This is managed by the runtime, you don't need to add it here.

Reply:
If someone has an intel GPY this is automatically downloaded by the runtime

Reply:
I misunderstood how it worked in flatpak, I've removed it